docs/email/components.html

```diff
@@ -0,0 +1,528 @@
+                                <code class="grid--cell stacks-code">.s-btn__white</code>
+                            </div>
+                        </td>
+                        <td class="va-middle px8" style="background-color: #242729;">
+                            <a class="s-btn s-btn__white" href="#" style="background: #ffffff; border: 1px solid #ffffff; box-shadow: 0 0 0 0 rgba(0,149,255,0); font-family: arial, sans-serif; font-size: 17px; line-height: 17px; color: #0077cc; text-align: center; text-decoration: none; padding: 13px 17px; display: block; border-radius: 4px;">
+                                Ask Question
+                            </a>
+                        </td>
+                        <td class="va-middle px8" style="background-color: #242729;">
+                            <a class="s-btn s-btn__white" href="#" style="background: rgba(255,255,255,.7); border: 1px solid rgba(255,255,255,.7); box-shadow: 0 0 0 0 rgba(0,149,255,0); font-family: arial, sans-serif; font-size: 17px; line-height: 17px; color: #0077cc; text-align: center; text-decoration: none; padding: 13px 17px; display: block; border-radius: 4px;">
+                                Ask Question
+                            </a>
+                        </td>
+                    </tr>
+                    <tr>
+                        <th scope="row" class="va-middle">Outlined</th>
+                        <td class="va-middle">
+                            <div class="grid gs4 fw-wrap">
+                                <code class="grid--cell stacks-code">.s-btn</code>
+                                <code class="grid--cell stacks-code">.s-btn__outlined</code>
+                            </div>
+                        </td>
+                        <td class="va-middle px8">
+                            <a class="s-btn s-btn__outlined" href="#" style="background: #ffffff; border: 1px solid #0077cc; font-family: arial, sans-serif; font-size: 17px; line-height: 17px; color: #0077cc; text-align: center; text-decoration: none; padding: 13px 17px; display: block; border-radius: 4px; white-space: nowrap;">
+                                Ask Question</span></a>
+                        </td>
+                        <td class="va-middle px8">
+                            <a class="s-btn s-btn__outlined" href="#" style="background: rgba(0,119,204,.05); border: 1px solid #0077cc; font-family: arial, sans-serif; font-size: 17px; line-height: 17px; color: #005999; text-align: center; text-decoration: none; padding: 13px 17px; display: block; border-radius: 4px; white-space: nowrap;">
+                                Ask Question
+                            </a>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+        <aside class="s-notice s-notice__info s-notice__outlined mt24" role="status" aria-hidden="false">
```

Comment:
Move notice outside of `div.stacks-preview--example`. Getting a connected box-shadow effect.

![image](https://user-images.githubusercontent.com/1544281/43609769-e4eb7e5e-9672-11e8-97c8-fcc6431a4a0e.png)